### PR TITLE
More shipment fixes and improvements

### LINKED ISSUE #6762: Shipment updates
A few more shipment updates / fixes (after https://github.com/IMA-WorldHealth/bhima/pull/6761 is merged):
- ~~Debug issue with updating packing list failing~~ (fixed in https://github.com/IMA-WorldHealth/bhima/pull/6761)
- Make sure creating/editing shipment containers cannot repeat labels
- ~~Add count of containers in shipments registry~~ (decided not to do this)

### PR DESCRIPTION
**Pull request by jmcameron**
Updates
      - Round weights for shipment document/manifest
      - Enforce uniqueness of shipment container labels

Closes https://github.com/IMA-WorldHealth/bhima/issues/6762

**TESTING** 
- Use bhima_test
- Create a shipment and try to create two containers with the same label
- Create/edit a container and try entering a negative weight.

### CODE REVIEW

File: client/src/modules/shipment/modals/edit-container.modal.js [hunk 1]
```diff
@@ -14,14 +14,16 @@ function ContainerEditModalController(Data, Containers, Notify, Instance) {
 
   vm.isCreate = Data.action === 'create';
   vm.existingContainer = Data.action === 'edit' && !(NOT_CREATED in Data.container);
-
+  vm.knownContainers = new Set(Data.known || []);
```
Comment (mbayopanda) [inline client/src/modules/shipment/modals/edit-container.modal.js]: 👍🏽

**Review by mbayopanda — COMMENTED** [review]
The feature is working fine I have just noticed that we can create another container with the same label with different case.

image

**Comment by jmcameron** [comment]
> The feature is working fine I have just noticed that we can create another container with the same label with different case.

fixed this issue.

**Review by mbayopanda — APPROVED** [review]
LGTM 👍🏽 

bors r+

**Pull request merged**